magicblock-config/tests/fixtures/11_everything-defined.toml

```diff
@@ -38,11 +38,10 @@ country-code = "US"
 claim-fees-interval-secs = 10
 
 [ledger]
-resume-strategy = "replay"
+resume-strategy = { variant = "replay", reset-slot = 100 }
```

Comment:
`variant `is really kind of abstract, (validator operators don't care that it is an enum _variant_). `kind` or `type` might be better.
Then also adjust the `LEDGER_RESUME_STRATEGY_VARIANT` accordingly